src/components/substance/SubstanceForm.vue

```diff
@@ -143,9 +143,7 @@ export default {
     ...mapState("qcLevel", { qcLevelList: "list" }),
 
     form: function() {
-      return this.type === "definedCompound"
-        ? this.getDefSubstanceForm
-        : this.getIndefSubstanceForm;
+      return this.$store.state.substance.form;
```

Comment:
This isn't pressing at all but by syncing with state this does in fact make edits directly to the state.  This is against [Vuex's recommendations](https://vuex.vuejs.org/guide/).

> 2. You cannot directly mutate the store's state. The only way to change a store's state is by explicitly **committing mutations**. This ensures every state change leaves a track-able record, and enables tooling that helps us better understand our applications.

I don't know if this is a deal breaker because a substance "form" is probably expected to change with user input, but if we were to instead load the form object locally it might be better for the sake of keeping with Vuex's recommendations.

Reply:
if I have the form loaded with a substance and I look at it in vuex in devtools, when I make a change to the field, nothing happens to the store? I would think it would if we were to use mapState to the form, but we are just returning the values from state into the computed property, so it isn't reactive, but as to whether or not this is a good convention, I wouldn't know. This is from the bottom of the page linked above
```
Using store state in a component simply involves returning the state within a computed property, because the store state is reactive. Triggering changes simply means committing mutations in component methods.
```
I think we're okay here, but let m know if you read it a different way

Reply:
Yeah.  The Vuex seem to work off the assumption that changes cannot be made without mutations, as such it only reloads the state on a change.  It's kind of a catch 22.  You can't see that you're changing the state because the dev tools think you won't be changing the state.

A way to illustrate that the state is actually changing is to post a reactive element on the page linking to the state.  For instance if you add `{{ $store.state.substance.form }}` onto the `{{repo_root}}/src/views/Substance.vue` right above your `<SubstanceSidebar />` you can get a reactive element linking to the form.

Edits to the below substance form will directly reflect in the reactive link to the state.  Once again, I'm not sure this is a deal breaker, but I hate going against such an explicit recommendation when I'm unsure what the side-effects will be.  

This behavior is also bound to be changed in #97 regardless.  If you want to just add a todo for that ticket, the substance form stuff may need a complete rework anyways.